frontend/hub/administration/repositories/RepositoryPage/RepositoryDetails.tsx

```diff
@@ -45,9 +45,9 @@ export function RepositoryDetails() {
       <PageDetail label={t('Repository name')}>{params.id}</PageDetail>
       <PageDetail label={t('Description')}>{repository.description || t('None')}</PageDetail>
       <PageDetail label={t('Retained version count')}>{repository.retain_repo_versions}</PageDetail>
-      <PageDetail label={t('Distribution')}>{distribution.name}</PageDetail>
+      <PageDetail label={t('Distribution')}>{distribution?.name || t('None')}</PageDetail>
       <PageDetail label={t('Repository URL')}>
-        <CopyCell text={distribution.client_url} />
+        {distribution?.client_url ? <CopyCell text={distribution.client_url} /> : t('None')}
```

Comment:
This one is showing as None sometimes after the creation.

Immediately after creation

<img width="1100" alt="image" src="https://github.com/ansible/ansible-ui/assets/9053044/4ccce385-3958-4aa8-9e85-a0c31892b256">


After page refresh

<img width="1084" alt="image" src="https://github.com/ansible/ansible-ui/assets/9053044/86c5fb5a-f540-49ed-a7e2-9230c54ada7c">

Reply:
It seems like this issue is being addressed by https://github.com/ansible/ansible-ui/pull/1532 just to inform you @ZitaNemeckova and @MilanPospisil .